components/service/glean/src/test/java/mozilla/components/service/glean/debug/GleanDebugActivityTest.kt

```diff
@@ -120,7 +120,7 @@ class GleanDebugActivityTest {
             category = "telemetry",
             lifetime = Lifetime.Application,
             name = "boolean_metric",
-            sendInPings = listOf("store1")
+            sendInPings = listOf("metrics")
```

Comment:
After this PR, sending completely arbitrary pings that don't have an entry in a `pings.yaml` is an error, so needed to rename these to a "known" ping.

Reply:
I actually like this, it's an extra protection against sending something invalid.

Reply:
Yeah.  It also makes it easier for new pings to require data review (you can just watch `pings.yaml`), which in turn will give the pipeline folks a heads up about new incoming docTypes.  Clients can't just start throwing a bunch of new stuff at the pipeline...